src/swapnaWJun20/RemoveTechnoArrayList.java

```diff
@@ -0,0 +1,28 @@
+package swapnaWJun20;
+
+import java.util.ArrayList;
+
+public class RemoveTechnoArrayList {
+
+	void removeWord(ArrayList<String> input, String toBeRemoved) {
+		for (int index = 0; index < input.size(); index++) {
+			if (input.contains(toBeRemoved)) {
+				input.remove(toBeRemoved);
+			}
+		}
+		System.out.println("Output : " + input);
+	}
```

Comment:
This is good but as we don't know the iteration, while is preferable.

Reply:
Thanks Anup. Code has updated as per the comment.